release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2872,6 +2872,48 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.15
+[id="ocp-4-17-15_{context}"]
+=== RHSA-2025:0876 - {product-title} {product-version}.15 bug fix, and security update advisory
+
+Issued: 04 February 2025
+
+{product-title} release {product-version}.15 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:0876[RHSA-2025:0876] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:0878[RHSA-2025:0878] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.15 --pullspecs
+----
+
+[id="ocp-4-17-15-bug-fixes_{context}"]
+==== Bug fixes
+
+// SME review
+* Previously, when you used the installation program to install a cluster in a Prism Central environment, the installation failed because a `prism-api` call that loads an {op-system} image timed out. This issue happened because the `prismAPICallTimeout` parameter was set to `5` minutes. With this release, the `prismAPICallTimeout` parameter in the `install-config.yaml` configuration file now defaults to `10 minutes. You can also configure the parameter if you need a longer timeout for a `prism-api` call. (link:https://issues.redhat.com/browse/OCPBUGS-49362[*OCPBUGS-49362*])
+
+// Ask SME about https://issues.redhat.com/browse/OCPBUGS-48745. SKip if no reply.
+
+* Previously, every time a subcription was reconciled, the {olm} catalog Operator requsted a full view of the catalog metadata from the catalog source pod of the subscription. These requests caused performance issues for the catalog pods. With this release, the {olm} catalog Operator now uses a local cache that is refreshed periodically and reused by all subscription reconciliations, so that the performance issue the performance issue for the catalog pods no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-48695[*OCPBUGS-48695*])
```

Comment:
```suggestion
* Previously, every time a subcription was reconciled, the {olm} catalog Operator requsted a full view of the catalog metadata from the catalog source pod of the subscription. These requests caused performance issues for the catalog pods. With this release, the {olm} catalog Operator now uses a local cache that is refreshed periodically and reused by all subscription reconciliations, so that the performance issue for the catalog pods no longer persists. (link:https://issues.redhat.com/browse/OCPBUGS-48695[*OCPBUGS-48695*])
```